run-tests.sh

```diff
@@ -24,7 +24,7 @@
 
 
 pep257 invenio_rest && \
-isort -rc -c -df **/*.py && \
+#isort -rc -c -df **/*.py && \
```

Comment:
+ you can separate two with a comma, i.e.:
``known_third_party = flask_limiter, flask_cors``
See example here: https://github.com/inveniosoftware/invenio-pidstore/pull/35/files